hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.tsx

```diff
@@ -17,44 +17,57 @@ import React from "react";
 
 export const TeamInfoTable = () => {
     return (
-        <Grid container direction="column" spacing={1} item md={6} xs={12}>
+        <Grid
+            container
+            direction="column"
+            spacing={1}
+            item
+            md={6}
+            xs={12}
+            style={{
```

Comment:
since you have a grid already, you don't need this margin, instead you should increase the spacing of your grid container.